xml/deployment_prep_aarch64_raspi.xml

```diff
@@ -0,0 +1,260 @@
+   first boot will be substantially slower than subsequent boots.
+  </para>
+  <para>
+   The process of writing the card image onto MicroSD media is described in the
+   Quick Start guide, here:
+   <link xlink:href="https://www.suse.com/documentation/sles-15/singlehtml/art-rpiquick/art-rpiquick.html#sec.rpi.installation"/>
+  </para>
+
+ </sect2>
+ <sect2>
+  <title>Installation from USB Media</title>
+  <para>
+   This method will be more familiar as it is the way &sls; is deployed
+   on other platforms. Installation can be performed from a removable USB
+   medium, such as a memory stick, onto a MicroSD card in the machine's internal
+   slot, or from a removable USB medium onto another USB medium, such as a
+   USB-connected hard disk.
+  </para>
+  <note>
+   <title>USB Bandwidth Limitations</title>
+   <para>
+    Note that the Ethernet, Wi-Fi and Bluetooth controllers on the &rpi; 3 are
+    all connected to the device's on-board USB 2 bus. Therefore an operating
+    system running from a disk attached via USB must share the total 480 Mbps
+    bandwidth of the USB controller. This will limit performance, and could
+    significantly impact network performance.
+   </para>
+  </note>
+  <para>
+   Installation on the &rpi; has some aspects that are unlike other
+   platforms.
+  </para>
+  <para>
+   The &rpi; differs from more conventional machines in two important ways:
+  </para>
+  <orderedlist>
+   <listitem>
+    <para>
+     The &rpi; hardware does not have any built-in firmware. Its firmware is
+     loaded from a special reserved FAT16 partition on the MicroSD card every
+     time that the machine is powered on. There is also no battery-backed clock
+     or non-volatile BIOS memory to hold configuration information.
+    </para>
+    <para>
+     This means there are no conventional settings to adjust for boot device
+     order, time and date, and so on.
+    </para>
+    <para>
+     Instead, the &rpi; has a very small amount of One-Time Programmable Memory
+     (OTP memory) which can be used to configure some settings, such as whether
+     the machine should attempt to boot from USB devices or over Ethernet. It is
+     only possible to program this by preparing a bootable MicroSD card and
+     placing certain configuration files on it.
+    </para>
+    <para>
+     This OTP memory is described on the &rpi; Foundation web site, here:
+     <link xlink:href="https://www.raspberrypi.org/documentation/hardware/raspberrypi/otpbits.mdd"/>
+    </para>
+    <note>
+     <title>Time Synchronization</title>
+     <para>
+      The lack of a Real Time Clock (RTC) also means that it is essential for
+      &rpi; devices to be configured to fetch the time from a network server by
+      &ntp;.
+    </para>
+    </note>
+   </listitem>
+   <listitem>
+    <para>
+     The primary processor on the &rpi; is the GPU, not the &arm; processor. It
+     is the GPU which initialises the system and loads the operating system. As
```

Comment:
s/GPU/Graphics Processing Unit (GPU)/g

Reply:
Done. I have added the name, for the avoidance of doubt.